src/gc/unix/gcenv.unix.cpp

```diff
@@ -678,6 +686,85 @@ bool GCToOSInterface::GetWriteWatch(bool resetState, void* address, size_t size,
     return false;
 }
 
+static size_t GetLogicalProcessorCacheSizeFromOS()
+{
+    size_t cacheSize = 0;
```

Comment:
This is just a copy of the CoreCLR PAL code. I hope we will eventually switch over to the custom GC PAL and get rid of all GC-specific functions in CoreCLR PAL.